app/views/RegisterView.js

```diff
@@ -84,13 +103,24 @@ export default class RegisterView extends LoggedView {
 		const {
 			name, email, password, username
 		} = this.state;
-		return name.trim() && email.trim() && password.trim() && username.trim() && isValidEmail(email);
+		return name.trim() && email.trim() && password.trim() && username.trim();
+	}
+
+	validEmail = () => {
```

Comment:
Can we change this function name?
This way we have `validEmail` and `isValidEmail` inside the same class.
Maybe `isValidEmailValidation` for the util.
What do you think?